gulpfile.js

```diff
@@ -0,0 +1,76 @@
+/*global require*/
+
+var WORK_OUT_FOLDER = './src/',
+    PROD = false;
```

Comment:
require this constants/variables from a config file, let's put in the root folder a json called `config.json` and require from

Reply:
I created the config.js file in the root folder.
The json format for these purposes is not very suitable in my opinion.

Reply:
fixed